query/transformation_test.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2015 Square Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package query
+
+import (
+	"math"
+	"testing"
+
+	"github.com/square/metrics/api"
+)
+
+func tagSetEquals(a, b api.TagSet) bool {
```

Comment:
Okay, implemented in this branch. I found two places where it had been defined; here and in `aggregate_test.go`. I'm not sure where else it might be.